### ISSUE TITLE
import module can't use in template

### ISSUE THREAD

**Issue by tonylu110** [vue-vine/vue-vine#168]
I import a module in `component.vine.ts` like
```ts
// component.vine.ts
import { foo } from 'foo.ts'

export function component() {
  return vine`
    <div :class="foo()"></div>
  `
}
```

```ts
// foo.ts
export function foo() {
  // more
  return ...
}
```

It have a error `$setup.foo is not a function`

**Comment by tonylu110** [comment]
in 0.1.34 can use

**Comment by ShenQingchuan** [comment]
Thanks for your investigation and very sorry to bring you this bug in our recent updates.

This is because in development mode,`@vue/compiler-dom` produces separated compile result just like this:

```js
const __sfc__ = {
  __name: 'App',
  setup(__props, { expose: __expose }) {
  __expose();

const msg = ref('Hello World!')

const __returned__ = { msg, ref, get foo() { return foo } }
Object.defineProperty(__returned__, '__isScriptSetup', { enumerable: false, value: true })
return __returned__
}

};
import { toDisplayString as _toDisplayString, createElementVNode as _createElementVNode, vModelText as _vModelText, withDirectives as _withDirectives, Fragment as _Fragment, openBlock as _openBlock, createElementBlock as _createElementBlock } from "vue"
function render(_ctx, _cache, $props, $setup, $data, $options) {
  return (_openBlock(), _createElementBlock(_Fragment, null, [
    _createElementVNode("h1", null, _toDisplayString($setup.msg), 1 /* TEXT */),
    _createElementVNode("div", {
      onClick: _cache[0] || (_cache[0] = $event => ($setup.foo()))
    }, "ddd"),
    _withDirectives(_createElementVNode("input", {
      "onUpdate:modelValue": _cache[1] || (_cache[1] = $event => (($setup.msg) = $event))
    }, null, 512 /* NEED_PATCH */), [
      [_vModelText, $setup.msg]
    ])
  ], 64 /* STABLE_FRAGMENT */))
}
__sfc__.render = render
__sfc__.__file = "src/App.vue"
export default __sfc__
```

And our Vine function component's context only contains a transformed version AST, which doesn't have a analyzable expression 'foo()' to extract that identifier, it's already a `$setup.foo()`.

So we need to obtain a parsed AST from `compilerDOM.parse( ... )` for `import-usage-check.ts` to walk through and analyze.

This issue should be fixed in **`v0.1.39`**
